pkg/sshutil/sshutil.go

```diff
@@ -172,7 +172,7 @@ func SSHArgs(instDir string, useDotSSH bool) ([]string, error) {
 		return nil, err
 	}
 	args = append(args,
-		"-l", u.Username, // guest and host have the same username, but we should specify the username explicitly (#85)
+		"-o", fmt.Sprintf("User=%s", u.Username), // guest and host have the same username, but we should specify the username explicitly (#85)
```

Comment:
Any specific reason for this change?

Reply:
`scp` doesn't support `-l` (that means `-l limit`, `Limit the used bandwidth` instead).  However, both `ssh` and `scp` does `-o User=xxx` because it's the format from `~/.ssh/config`.